hazel/tools/ghc.bzl

```diff
@@ -0,0 +1,28 @@
+load("@bazel_tools//tools/cpp:lib_cc_configure.bzl", "get_cpu_value")
+
+default_ghc_workspaces = {
```

Comment:
What is `@ghc`? Are these good defaults, are they correct?

Reply:
These defaults expect `rules_haskell` _users_ to define a `@ghc` workspace. I think we should rather use the default names for bindists.